src/content/config.ts

```diff
@@ -26,9 +26,19 @@ const events = defineCollection({
                     end: z.date().optional(),
                 })
                 .optional(),
-            locationName: z.string().optional(),
-            locationURL: z.string().url().or(z.string().startsWith('#')).or(z.array(z.string().url())).optional(),
-            locationLatLng: z.array(z.number(), z.number()).optional(),
+            locations: z.array(
+                z.object({
+                    locationName: z.string().optional(),
+                    locationURL: z.string().url().or(z.string().startsWith('#')).or(z.array(z.string().url())).optional(),
+                    locationLatLng: z.array(z.number(), z.number()).optional(),
+                    // Fields more relevant for online events
+                    slides: z.array(z.string()).optional(),
+                    doi: z.array(z.string()).optional(),
+                    youtube: z.string().optional(),
+                    biliBili: z.string().optional(),
+                    zoom: z.string().optional(),
```

Comment:
Maybe I misunderstood and these should be added just as items in the `locationURL` array?

Reply:
hmm, I think we use locationURL quite often for venue URLs, so might be fine to keep them seperated

Reply:
yeah, we had something like that for the old website. it's a bit of a pain to cover all different urls, but it will be mostly youtube, so maybe it's fine? 🤔

Reply:
In the end I went a bit further, a small summary:
- Locations is an array now
- Locations can have their own links (useful for the event's venue link)
- There is a new separate `links` array which represent links for the event in general, here some custom icons are added for youtube, github, bilibili, etc...

Example:

<img width="1512" alt="Screenshot 2024-03-18 at 17 06 38" src="https://github.com/nf-core/website/assets/32884354/ae0d7538-9e66-4554-8d92-39e98ee87596">

Reply:
very nice!
need just to play with the layout a bit to make the columns more balanced (and maybe center the event export button).
because we have the icons, I don't think we need the bullets for the links list.
Thinking further: maybe we just have the icons and don't show the links at all (because they are usually quite ugly)(but make them more discoverable, so that's something we need to try out/discuss)

Reply:
I actually was thinking that the export button should be placed next to the "Edit" button, or in the same corner to say the least. Moving link icons below the event dates and leaving the right column for the locations might balance everything a bit better, I will try tomorrow!